GLEW build failure on Ubuntu

**Host Environment**
- OS: Linux (Ubuntu 20.04 via Github Actions)
- Compiler: GCC and Clang

**To Reproduce**
Steps to reproduce the behavior:
(in Github Actions)
' git submodule add https://github.com/microsoft/vcpkg vcpkg/'
'pushd vcpkg'
'./bootstrap-vcpkg.sh -disableMetrics'
'./vcpkg install catch2 glew glfw3 --triplet x64-linux'
(permalink to yaml file https://github.com/Journeyman-dev/TUIC/blob/b97438671742a044c69f11a2814b5ef60fd37982/.github/workflows/Tests.yml)
(actions report https://github.com/Journeyman-dev/TUIC/actions/runs/974265097)

**Failure logs** 
(logs extracted from Github Actions using actions/archive)
```
CMake Error at /home/runner/work/TUIC/TUIC/vcpkg/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find OpenGL (missing: OPENGL_opengl_LIBRARY OPENGL_glx_LIBRARY
  OPENGL_INCLUDE_DIR)
Call Stack (most recent call first):
  /home/runner/work/TUIC/TUIC/vcpkg/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /home/runner/work/TUIC/TUIC/vcpkg/downloads/tools/cmake-3.20.2-linux/cmake-3.20.2-linux-x86_64/share/cmake-3.20/Modules/FindOpenGL.cmake:443 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  /home/runner/work/TUIC/TUIC/vcpkg/scripts/buildsystems/vcpkg.cmake:771 (_find_package)
  CMakeLists.txt:40 (find_package)
```

**Additional context**
it is apparent that vcpkg recognizes that opengl[core] is needed for glew to work and downloads it as a dependency:
```
Run ./vcpkg install catch2 glew glfw3 --triplet x64-linux
Computing installation plan...
The following packages will be built and installed:
    catch2[core]:x64-linux -> 2.13.6
    glew[core]:x64-linux -> 2.1.0#10
    glfw3[core]:x64-linux -> 3.3.4
  * opengl[core]:x64-linux -> 0.0#8
  ```
 It should be noted that  opengl[core] is not needed for glfw3 or catch2, so it must be installing it for glew unless there is a mistake in one of those two packages. 
 
 Despite the fact that vcpkg successfully installs opengl[core], it fails to link it with glew correctly. Also note that instead of using the built in vcpkg install that ships with GitHub Actions server, I cloned it as a submodule to ensure that I am using the latest version of vcpkg. As you can see from the Actions report, this error does not occur on Windows and Mac operating systems.

--- a/ports/glew/CONTROL
+++ b/ports/glew/CONTROL
@@ -1,6 +1,0 @@
-Source: glew
-Version: 2.1.0
-Port-Version: 10
-Description: The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library.
-Homepage: https://github.com/nigels-com/glew
-Build-Depends: opengl

--- a/ports/glew/portfile.cmake
+++ b/ports/glew/portfile.cmake
@@ -1,5 +1,5 @@
 if(VCPKG_TARGET_IS_LINUX)
-    message(WARNING "${PORT} currently requires the following libraries from the system package manager:\n    libxmu-dev\n    libxi-dev\n    libgl-dev\n\nThese can be installed on Ubuntu systems via apt-get install libxmu-dev libxi-dev libgl-dev.")
+    message(WARNING "${PORT} requires the following libraries from the system package manager:\n    libxmu-dev\n    libxi-dev\n    libgl-dev\n\nThese can be installed on Ubuntu systems via apt-get install libxmu-dev libxi-dev libgl-dev.")
 endif()
 
 # Don't change to vcpkg_from_github! The sources in the git repository (archives) are missing some files that are distributed inside releases.

--- a/ports/glew/portfile.cmake
+++ b/ports/glew/portfile.cmake
@@ -1,3 +1,7 @@
+if(VCPKG_TARGET_IS_LINUX)
+    message(WARNING "${PORT} currently requires the following libraries from the system package manager:\n    libxmu-dev\n    libxi-dev\n    libgl-dev\n\nThese can be installed on Ubuntu systems via apt-get install libxmu-dev libxi-dev libgl-dev.")
+endif()
+
 # Don't change to vcpkg_from_github! The sources in the git repository (archives) are missing some files that are distributed inside releases.
 # More info: https://github.com/nigels-com/glew/issues/31 and https://github.com/nigels-com/glew/issues/13
 vcpkg_download_distfile(ARCHIVE

--- a/ports/glew/vcpkg.json
+++ b/ports/glew/vcpkg.json
@@ -1,0 +1,13 @@
+{
+  "name": "glew",
+  "version-string": "2.1.0",
+  "port-version": 11,
+  "description": "The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library.",
+  "homepage": "https://github.com/nigels-com/glew",
+  "dependencies": [
+    {
+      "name": "opengl",
+      "platform": "windows"
+    }
+  ]
+}

--- a/ports/glew/vcpkg.json
+++ b/ports/glew/vcpkg.json
@@ -5,9 +5,6 @@
   "description": "The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library.",
   "homepage": "https://github.com/nigels-com/glew",
   "dependencies": [
-    {
-      "name": "opengl",
-      "platform": "windows"
-    }
+    "opengl"
   ]
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2290,7 +2290,7 @@
     },
     "glew": {
       "baseline": "2.1.0",
-      "port-version": 10
+      "port-version": 11
     },
     "glfw3": {
       "baseline": "3.3.4",
@@ -3539,8 +3539,8 @@
     "libsigcpp-3": {
       "baseline": "3.0.3",
       "port-version": 1
-    }, 
-	"libsmb2": {
+    },
+    "libsmb2": {
       "baseline": "2021-04-29",
       "port-version": 0
     },

--- a/versions/g-/glew.json
+++ b/versions/g-/glew.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "f0d1692409b7a934c21fc44652ffac9ec508c0a9",
+      "git-tree": "622e27b2a746c088f0acd2f98445c0968f485a69",
       "version-string": "2.1.0",
       "port-version": 11
     },

--- a/versions/g-/glew.json
+++ b/versions/g-/glew.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "f0d1692409b7a934c21fc44652ffac9ec508c0a9",
+      "version-string": "2.1.0",
+      "port-version": 11
+    },
+    {
       "git-tree": "c87d7f619c69630fa4d1bd0bf3767f0d31ef22d6",
       "version-string": "2.1.0",
       "port-version": 10

